src/zipObjWith.js

```diff
@@ -0,0 +1,31 @@
+import { curryN, pipe, apply, map, fromPairs, zip } from 'ramda';
+
+/**
+ * Creates a new object out of a list of keys and a list of values by applying the function
+ * to each equally-positioned pair in the lists.
+ * Key/value pairing is truncated to the length of the shorter of the two lists.
+ *
+ * @func zipObjWith
+ * @memberOf RA
+ * @category Object
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.22.0|v2.22.0}
+ * @sig Functor f => (v, k) => [c, d] -> [k] -> [v] -> { c: d }
```

Comment:
Align this signature with `unzipObjWith`. This function is not a functor as I pointed out in that PR.